tasks/main.yml

```diff
@@ -0,0 +1,117 @@
+---
+
+- name: check if passwords are specified
+  assert:
+    that:
+      - database_password is truthy
+      - database_root_password is truthy
```

Comment:
Why set defaults for these in `defaults/main.yml` if you require different values anyway? You could just remove them from `defaults/main.yml`, couldn't you?

Reply:
True, removed from defaults in https://github.com/elan-ev/opencast_mariadb/pull/1/commits/9a27f517a23058c9d76b0eb2f441f393e63e0765